src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/XFIssue/Issue3019.cs

```diff
@@ -12,16 +12,13 @@ public Issue3019(TestDevice testDevice) : base(testDevice)
 
 	public override string Issue => "Grouped ListView Header empty for adding items";
 
-	// [Test]
-	// [Category(UITestCategories.ListView)]
-	// [FailsOnIOS]
-	// public void MakeSureListGroupShowsUpAndItemsAreClickable()
-	// {
-	// 	App.WaitForElement("Group 1");
-
-	// 	App.Tap(x => x.Marked("Grouped Item: 0"));
-	// 	App.Tap(x => x.Marked("Grouped Item: 1"));
-	// 	App.Tap(x => x.Marked("Grouped Item: 1 Clicked"));
-
-	// }
+	[Test]
+	[Category(UITestCategories.ListView)]
+	public void MakeSureListGroupShowsUpAndItemsAreClickable()
```

Comment:
This test is failing on Android:
<img width="1145" alt="image" src="https://github.com/user-attachments/assets/94f176da-42c6-43e6-8fdd-d6bb40a2fa41">

Reply:
Fixed the test failure and committed the changes.
<img width="382" alt="image" src="https://github.com/user-attachments/assets/be45d91d-b813-4196-9474-0c9cc688233a">